onnx/defs/nn/defs.cc

```diff
@@ -278,17 +281,17 @@ and computes the output.)DOC";
                         AttributeProto::INTS, OPTIONAL);
             schema.Attr("strides",
                         "stride along each axis.",
-                        AttributeProto::INTS, OPTIONAL);
```

Comment:
Could be optional with "1" stride in all axes?

Reply:
we could, but the problem is the len of the list of strides.
we can't know the default len of the list of stride values.
So, two solution : 
- To make this attributes required
- To make this attribute optional, without any default value, but in that case, modify the doc to say that back-ends have to use stride of 1 for each spacial axis of input

Reply:
I'd pick up the option 2 here.

Reply:
Ok, so i will make the change to fit option 2.
So the PR will not be BC